framework/src/media/utils/MediaUtils.cpp

```diff
@@ -462,5 +462,125 @@ bool header_parsing(unsigned char *buffer, unsigned int bufferSize, audio_type_t
 	}
 	return true;
 }
+
+bool write_little_endian(FILE *fp, unsigned int num, unsigned short byte)
+{
+	unsigned char buf;
+	int ret = 0;
+	for (int i = 0; i < byte; ++i) {
+		buf = num & 0xff;
+		ret += fwrite(&buf, sizeof(char), 1 ,fp);
+		num >>= 8;
+	}
+	if (ret != byte) {
+		medvdbg("file write failed error\n");
+		return false;
+	}
+	return true;
+}
+
+bool create_wav_header(FILE *fp)
+{
+	unsigned char *buffer;
+	buffer = (unsigned char *)malloc(WAVE_HEADER_LENGTH);
+	if (buffer == NULL) {
+		medvdbg("fail to malloc buffer\n");
+		if (fclose(fp) == 0) {
+			fp = nullptr;
+		}
+		return false;
+	}
+
+	memset(buffer, 0, WAVE_HEADER_LENGTH);
+	fwrite(buffer, sizeof(unsigned char), WAVE_HEADER_LENGTH, fp);
+	free(buffer);
+	return true;
+}
+
+bool write_wav_header(FILE *fp, unsigned int channel, unsigned int sampleRate, audio_format_type_t pcmFormat, unsigned int fileSize) {
+	/**
+	*wave header is
```

Comment:
I like this, but please clean here(blank).
``
 /*
   * blah blah
   */

Reply:
i modify